cmd/stack_package.go

```diff
@@ -242,7 +247,7 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 				Debug.Log("sourceDir is: ", sourceDir)
 
 				// create name for the tar files
-				versionedArchive := filepath.Join(devLocal, stackID+".v"+stackYaml.Version+".templates.")
+				versionedArchive := filepath.Join(devLocal, repoDirName+"."+stackID+".v"+stackYaml.Version+".templates.")
```

Comment:
I think we discussed that the CI scripts would prepend the repo name with the `add-to-repo --release-url` option.
Does that sound familiar?